data/thebarshablog.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Barsha Das",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Founder of **The CookieButton Project**|Co-organizer of **KubeHuddle Toronto 2023** 🇨🇦",
+  "avatar": "https://github.com/thebarshablog.png",
+  "links": [
+    {
+      "name": "Join Me In Building My Personal Website",
+      "url": "https://thebarshablog.github.io/",
+      "icon": "fa-solid fa-browser"
```

Comment:
These are not valid icons, please select only 1 icon for each link from the available icons https://linkfree.eddiehub.io/icons